instrumentation/opentelemetry-instrumentation-tornado/src/opentelemetry/instrumentation/tornado/__init__.py

```diff
@@ -235,11 +235,12 @@ def _get_attributes_from_request(request):
         SpanAttributes.HTTP_TARGET: request.path,
     }
 
-    if request.host:
-        attrs[SpanAttributes.HTTP_HOST] = request.host
-
     if request.remote_ip:
-        attrs[SpanAttributes.NET_PEER_IP] = request.remote_ip
+        attrs[SpanAttributes.HTTP_CLIENT_IP] = request.remote_ip
+        if getattr(request, "_orig_remote_ip"):
+            attrs[SpanAttributes.NET_PEER_IP] = request._orig_remote_ip
+        else:
+            attrs[SpanAttributes.NET_PEER_IP] = request.remote_ip
```

Comment:
nit: Isn't this already captured above as HTTP client IP? May be we can skip it here and only capture when `_orig_remote_ip` is present.

Reply:
Sure I don't mind. `_orig_remote_ip` is always set in practice, in current versions.